gui/services/models.py

```diff
@@ -318,8 +318,7 @@ class AFP(Model):
         ),
         max_length=255,
         blank=True,
-        choices=list(choices.IPChoices()),
-        default='',
+        null=True,
```

Comment:
If you change it to null it will require a migration. If it is not needed it should be changed to default='' instead of null=True.